docs/articles/faq/README.md

```diff
@@ -283,3 +283,21 @@ According to users' feedback, the following CI systems work best with TestCafe:
 * [TravisCI](../documentation/guides/continuous-integration/travis.md)
 * [CircleCI](../documentation/guides/continuous-integration/circleci.md)
 * [AppVeyor](../documentation/guides/continuous-integration/appveyor.md)
+
+### 'Browser can't open the page: can't establish a secure connection to the server'
+
+If the tested website is served via OpenVPN, TestCafe may be unable to detect an IP address from where to retrieve the website. In this case, the following error occurs:
+
+```text
+<browsername> can't open the page "https://localhost:<port>/browser/connect/<windowid>"
```

Comment:
> https://localhost

Why is `localhost` in this message?
I thought that TestCafe tries to open `localhost` only when you specify `--hostname localhost` explicitly (an approach that you propose as a solution). Otherwise, it uses an IP.

Is this behavior specific to the VPN issue (that TestCafe uses `localhost` by default)?

Reply:
> Otherwise, it uses an IP.

Seems like sometimes TestCafe picks an IP that belongs to the VPN connection (which leads to the aforementioned error).